stock_orderpoint_manual_procurement/wizards/make_procurement_orderpoint.py

```diff
@@ -125,3 +113,34 @@ def onchange_uom_id(self):
         for rec in self:
             rec.qty = rec.orderpoint_id.product_uom._compute_quantity(
                 rec.orderpoint_id.procure_recommended_qty, rec.uom_id)
+
+    @api.model
+    def default_get(self, fields):
+        res = super(MakeProcurementOrderpointItem, self).default_get(fields)
+        orderpoint_obj = self.env['stock.warehouse.orderpoint']
+        orderpoint_ids = self.env.context['active_ids'] or []
+        active_model = self.env.context['active_model']
+
+        if not orderpoint_ids:
+            return res
+        assert active_model == 'stock.warehouse.orderpoint', \
```

Comment:
Please change to UserError here

Reply:
Comment attended, not sure if they are right.